root/layout/components/MergeHelper.js

```diff
@@ -61,15 +61,16 @@ const MergeHelper = ({merger}: Props): React.Element<'div'> => {
           }
         </p>
         <div className="buttons" style={{display: 'table-cell'}}>
-          {merger.ready_to_merge &&
+          {merger.ready_to_merge ? (
```

Comment:
Maybe @MonkeyDo could chime in on this too - 
My specific question was about doing `condition && statement` when the statement had a side effect, such as in the case of `entityProperties.aliases && inc.push('aliases');`

In jsx, I've always used `condition && <something>` when deciding if something should be output conditionally. So I'm not sure if it makes sense to use a ternary operator here. I guess the && syntax is pretty well accepted as a way of rendering react.

That is,
* in code use an if statement if the body performs an action.
* use a ternary if you want to choose between 2 values
* use && in jsx when choosing to conditionally display something

Reply:
I agree, in JSX the format `{ myCondition && <render something>}` is preferable.

Falsy values will be ignored by React and nothing will be rendered.

Reply:
Eww :) Well, we don't use that format almost anywhere, so I'd much prefer to be consistent. I also find it confusing and ugly, but that's a secondary issue.

Reply:
If you use the ternary operator everywhere in JSX then I guess that's another thing, and being consistent would be better. so feel free to proceed!

Reply:
IIRC we were only bit by the `&&` a few times when `myCondition` was the number `0` (since React would render that).  We do use the ternaries pretty consistently at this point.